pkg/api/validation/validation.go

```diff
@@ -1481,7 +1483,15 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 			allErrs = append(allErrs, field.Invalid(specPath.Child("storageClassName"), pv.Spec.StorageClassName, msg))
 		}
 	}
-
+	if pv.Spec.VolumeMode != nil && *pv.Spec.VolumeMode == api.PersistentVolumeBlock && !utilfeature.DefaultFeatureGate.Enabled(features.BlockVolumeSupport) {
+		allErrs = append(allErrs, field.Forbidden(specPath.Child("volumeMode"), "PersistentVolume volumeMode is disabled by feature-gate"))
+		return allErrs
+	}
+	if pv.Spec.VolumeMode != nil && len(*pv.Spec.VolumeMode) > 0 {
```

Comment:
This implies that "" is valid value. I think that only "Filesystem" and "Block" should be allowed.

Reply:
Remove the check for len().  Then it will error if volume mode is ""